frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -227,15 +227,15 @@ export const createResources = async (
   dryRun: boolean = false,
 ): Promise<K8sResourceKind[]> => {
   const {
+    name,
+    application: { name: applicationName },
```

Comment:
can fetch `image name` here as well in  `isi: { ports }` and pass it to knative service. It'll make it consistent with `import-submit-utils` 

`const imageStreamName = imageStream && imageStream.metadata.name;`

Reply:
We don't fetch the `imageStreamName` for deploy Image anywhere. So it doesn't make sense that we do it specifically for Knative and not for others.